api-docs.js

```diff
@@ -103,43 +103,38 @@ const apiDoc = {
     '/submit-job': {
```

Comment:
Can you add the script used to generate the json and yaml files? I agree that it is easier to have those docs committed but I'm wondering how we protect from updating the `api-docs.js` and forgetting to update the generated files. Any suggestion?

Reply:
I generated them manually using the `swagger editor`.

From what I understand, the only reason we have a `.js` file in this project is to be able to add comments to the docs

What I suggest is removing the api-docs.js file and keeping only `docs/apidocs.yml` as yaml supports comments.

In other javascript projects, we could parse the yaml file to use it on javascript (for a http route or something like this).

What do you think?

Reply:
Ok. I agree that keeping only `docs/apidocs.yml` is better and more handy.

Reply:
@msbrogli Done!